test/source/browser/controllable.ts

```diff
@@ -624,6 +624,18 @@ export class ControllablePage extends ControllableBase {
       }
     }
   }
+
+  public getFromLocalStorage = async (keys: string[]): Promise<any> => {
```

Comment:
Nice!! I did not know this was possible. This is an amazingly elegant solution.

Reply:
Can we use https://www.npmjs.com/package/googleapis package to access gmail api?

Reply:
Sure, I'm pretty liberal with adding deps to the test infrastructure.